locales/es.json

```diff
@@ -11,33 +11,33 @@
+   "The web3 object and the interfaces for the deployed contracts and their methods are also available": "El objeto web3 y las interfaces para los contratos desplegados y sus métodos también están disponibles",
    "create a working dapp with a SimpleStorage contract": "Crear una dapp funcional con un contrato SimpleStorage",
-   "filename to output logs (default: none)": "fichero/archivo para salida de los logs (predefinido: none)",
+   "filename to output logs (default: none)": "archivo de escritura de logs (Por defecto: none)",
    "level of logging to display": "nivel de log para mostrar",
-   "deploy and build dapp at ": "Publica los contractos y construye la aplicación en ",
-   "port to run the dev webserver (default: %s)": "puerto para correr el servidor web para desarrollo (default: %s)",
-   "host to run the dev webserver (default: %s)": "host para correr el servidor web para desarrollo (default: %s)",
+   "deploy and build dapp at ": "Despliega los contratos y construye la aplicación en ",
+   "port to run the dev webserver (default: %s)": "puerto para ejecutar el servidor web para desarrollo (Por defecto: %s)",
+   "host to run the dev webserver (default: %s)": "host para ejecutar el servidor web para desarrollo (Por defecto: %s)",
    "disable the development webserver": "desactiva el servidor web para desarrollo",
    "simple mode, disables the dashboard": "modo simple, desactiva el dashboard",
    "no colors in case it's needed for compatbility purposes": "Sin colores, en caso de ser necessario para compatibilidad con la terminal",
-   "filename to output logs (default: %s)": "fichero/archivo para los logs (predefinido: %s)",
-   "run dapp (default: %s)": "ejecutar la dapp (applicación decentralizada) (predefinido: %s)",
-   "Use a specific ethereum client or simulator (supported: %s)": "Usa un cliente o un simulador de ethereum específico (soportado: %s)",
-   "run blockchain server (default: %s)": "ejecuta un nodo de blockchain (predefinido: %s)",
-   "run a fast ethereum rpc simulator": "ejecuta un simulador RPC de ethereum",
-   "use testrpc as the rpc simulator [%s]": "usa testrpc como simulator de rpc [%s]",
-   "port to run the rpc simulator (default: %s)": "puerto para ejecutar un simulador de rpc (predefinido: %s)",
-   "host to run the rpc simulator (default: %s)": "host para ejecutar un servidor de rpc (predefinido: %s)",
-   "number of accounts (default: %s)": "número de cuentas (predefinido: %s)",
-   "Amount of ether to assign each test account (default: %s)": "Cantidad de éter para asignar a cada cuenta de prueba (predefinido: %s)",
-   "custom gas limit (default: %s)": "límite de gas (predefinido: %s)",
-   "run tests": "ejecutar las pruebas",
-   "resets embarks state on this dapp including clearing cache": "restablece el estado de Embark en esta dapp, incluida la eliminación de la memoria caché",
+   "filename to output logs (default: %s)": "archivo de escritura de logs (Por defecto: %s)",
```

Comment:
-Line 26 should be: “nombre de archivo de los registros de salida (por defecto: %s)”